microbench/bfs/src/sssp.cpp

```diff
@@ -12,9 +12,7 @@ void bfs::updateData(std::uint64_t val, pando::GlobalRef<std::uint64_t> ref) {
     if (val >= temp) {
       break;
     }
-  } while (!pando::atomicCompareExchange(&ref, pando::GlobalPtr<std::uint64_t>(&temp),
-                                         pando::GlobalPtr<std::uint64_t>(&val),
-                                         std::memory_order_relaxed, std::memory_order_relaxed));
+  } while (!pando::atomicCompareExchange(&ref, temp, val));
```

Comment:
why is this change required if the below is what is causing the problem?

Reply:
I mean it certainly is not necessary but it saves us a bunch of transformation. Now I don't have to transform a pointer to uint64_t to GlobalPtr and dereference it within the atomicCompareExchange function. I stick with whatever uint64_t value for the entire function.